AirCasting/ABConnector/BluetoothManager.swift

```diff
@@ -134,7 +142,10 @@ extension BluetoothManager: CBPeripheralDelegate {
 
     func peripheral(_ peripheral: CBPeripheral, didUpdateValueFor characteristic: CBCharacteristic, error: Error?) {
         if MEASUREMENTS_CHARACTERISTIC_UUIDS.contains(characteristic.uuid) {
-            let parsedMeasurement = parseData(data: characteristic.value!)
+            if let parsedMeasurement = parseData(data: characteristic.value!) {
```

Comment:
Seams that there is no documentation stating that the `value` will be always not nil so this should be validated with a standard guard. it may be unsafe to force unwrap it.